docs/howto/operate/manual-deploy.md

```diff
@@ -1,52 +1,74 @@
-# Manually deploy a hub
+# Set up and use the the deployment scripts locally
 
-The [`deploy.py`](https://github.com/2i2c-org/pilot-hubs/blob/master/deploy.py) script is a cmd utility built to help deploy one or all the hubs to their Kubernetes clusters, but  also to build the user image. Checkout the [](../configure/update-env.md) section on how to use the deploy script to update the user environment.
+While deploys generally go through our GitHub Actions workflow, sometimes you
+need to deploy from your laptop - primarily when testing changes on staging or
+actively fixing an outage. This isn't ideal, but this is where we are now.
 
-## Deploy script arguments and usage
-Running:
-```
-python3 deploy.py deploy --help
-```
-Will provide information about the `deploy.py` deploy options:
+Our deployment scripts live in the [`deploy/`](https://github.com/2i2c-org/pilot-hubs/blob/master/deploy.py)
```

Comment:
```suggestion
Our deployment scripts live in the [`deploy/` folder](https://github.com/2i2c-org/pilot-hubs/blob/master/deploy.py)
```

Reply:
That URL link should be https://github.com/2i2c-org/pilot-hubs/blob/master/deployer with the latest change.

Reply:
Done!